.github/workflows/pull-request.yaml

```diff
@@ -0,0 +1,29 @@
+name: pull request
+on: [pull_request]
+jobs:
+  rust:
+    runs-on: ubuntu-latest
+    env:
+      CARGO_INCREMENTAL: 0
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+          npm install ganache-cli
+          node_modules/.bin/ganache-cli --networkId 5777 --gasLimit 10000000 --gasPrice 0&
```

Comment:
nit: for debugging purposes, is it possible to pipe the output into a file and cat it at the end?

Reply:
should be possible but when would we print it? the job stops when something fails so just printing at the end wouldn't help in that case. if I see an easy to achieve this i'll do it but if not it isn't worth it to make the build script more complicated.